src/node.js

```diff
@@ -22,6 +22,50 @@
 
     EventEmitter.call(process);
 
+    process.on('warning', (warning) => {
+      if (process.noProcessWarnings) return;
+      if (process.traceProcessWarnings)
+        console.error(warning.stack);
+      else
+        console.error(`${warning.name}: ${warning.message}`);
+    });
+    process.on('deprecation', (warning) => {
+      if (process.noDeprecation) return;
+      if (process.traceDeprecation)
+        console.error(warning.stack);
+      else
+        console.error(`${warning.name}: ${warning.message}`);
```

Comment:
This is certainly possible. The default `Error.prototype.toString()` outputs `${name}: ${message}` anyway, so we could replace `console.error(`${warning.name}: ${warning.message}`);` with `console.error(warning.toString())`.